packages/clay-management-toolbar/src/Search.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IProps extends React.FormHTMLAttributes<HTMLFormElement> {
+	onlySearch?: boolean;
+
+	showMobile?: boolean;
```

Comment:
Why is this exactly for mobile if the class name is just `show`? Maybe adding some docs to the interfaces might help

Reply:
Ideally it was meant to be clear, but I see it wasn't 😅... In the mobile view the search will disappear, so you have to add the show class to show the search in the mobile view. I will take care of adding docs in these interfaces.